### DISCUSSION TITLE
Delta Lake connector query change data feed entries of the table like using Spark

### DISCUSSION THREAD

**Discussion by lamheihung** [trinodb/trino#17095] [Q&A] [answered]
Trino version: `413`

Starting from version `408`, [Trino](https://trino.io/docs/current/release/release-408.html#delta-lake-connector) adds support for creating tables with the Trino `change_data_feed_enabled` table property.

I already have some delta table and data in AWS S3, which is built from using PySpark, with change data feed enabling. When I create the table through Trino Delta connector, I set this property to `True`. When I query the table, it will return latest image of each record by its defined key. It seems no difference comparing to set this property to `False`.

If you use Spark SQL to read the delta table, you can configure the read option `readChangeFeed` and control the result, which is the latest image or all the historical change log of the data.

My question is, how to write the SQL statement in Trino, such that I can perform the similar reading control just like setting `readChangeFeed` in PySpark?

Thanks in advanced.
Jonathan Lam

**Comment by findinpath** [comment, marked as answer]
Check out https://trino.io/docs/current/connector/delta-lake.html#table-changes
